flows/nodes/http.md

```diff
@@ -39,3 +41,11 @@ Body: <15>1 2022-03-15T23:27:16.315639Z web1.floren.lan webserver 0 - [gw@1 host
 ```
 
 Note the headers; the `Accept-Encoding` and `User-Agent` headers are automatically set by the search agent.
+
+### Example With Decode Body
+
+The HTTP node can return data of any type, by default the node will store the response data as an array of bytes; when working with binary data or grabbing a remote file to us as a resource this is the most useful form.  Many use cases for the HTTP node may want to natively process an HTTP response so that the flow can more easily handle the response data; examples may include better display of text data or accessing fields in a JSON response.
```

Comment:
```suggestion
The HTTP node can return data of any type. By default, the node will store the response data as an array of bytes; when working with binary data or grabbing a remote file to use as a resource, this is the most useful form.  Other use cases for the HTTP node may want to natively process an HTTP response so that the flow can more easily handle the response data; examples may include better display of text data or accessing fields in a JSON response.
```